metadata.yaml

```diff
@@ -44,6 +44,12 @@ requires:
   openfga:
     interface: openfga
     limit: 1
+  oauth:
+    interface: oauth
+    limit: 1
+    description: |
+      Receive oauth server's info and a set of client credentials.
+      This relation can be used to integrate grafana with an oAuth2/OIDC Provider.
```

Comment:
possible typo: it mentions Grafana, should it mention Admin UI?

Reply:
oops